src/CommunityCommons/javasource/communitycommons/XPath.java

```diff
@@ -33,301 +33,372 @@
 import com.mendix.systemwideinterfaces.core.IMendixIdentifier;
 import com.mendix.systemwideinterfaces.core.IMendixObject;
 
-public class XPath<T>
-{
-	/** Build in tokens, see: https://world.mendix.com/display/refguide3/XPath+Keywords+and+System+Variables
-	 * 
+public class XPath<T> {
```

Comment:
The curly brace is on the same line here. That does not match with the generated actions. Should we still fix that?